test/functional/plugin/lsp_spec.lua

```diff
@@ -6121,6 +6121,23 @@ describe('LSP', function()
     end)
   end)
 
+  describe('vim.lsp.start', function()
+    it('can launch when an existing client has no workspace_folders', function()
+      exec_lua(create_server_definition)
+      exec_lua [[
```

Comment:
```suggestion
      exec_lua(function()
```

We can pass Lua functions to exec_lua now.